src/components/namestable/nametable-section.component.tsx

```diff
@@ -58,6 +63,14 @@ export function NameTableSection({ dataRow, workerType }: NameTableCellOptions):
     }
   }
 
+  function isClickable(flag: boolean): string {
+    if (flag) {
+      return "pointer";
+    }
+
+    return "default";
+  }
+
```

Comment:
tą funkcję koniecznie trzeba inaczej nazwać, bo jakbym zobaczył isClickable to bym pomyślał, że zwraca boolean.
isClickable to powinna być nazwa argumentu
a funkcja może cursorType, albo coś w tym stylu
dodatkowo tutaj bardzo ładnie można użyć ternary expression ;) ( isClickable ? "pointer" : "default" )